README.md

```diff
@@ -41,10 +41,20 @@ more data or running the full test suite, use:
 ``` r
 pak::pak(
   "steno-aarhus/osdc",
-  dependencies = c("all", "Config/Needs/tests", "Config/Needs/data")
+  dependencies = c("all", "Config/Needs/data")
```

Comment:
Why did you remove the tests here? 
On L39 above it says "[...] or running the full test suite, use:". Is the `tests` not needed for that?

Reply:
There's a CRAN check issue that I am trying to debug and I want to test if this will fix it. It might be brought back depending on the issue.